rlclientlib/CMakeLists.txt

```diff
@@ -1,157 +1,160 @@
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")

```

Comment:
Line damage in this file, please avoid it as no one can tell whether there was a change or not.

Reply:
thanks, fixed